102 summarize package scope reference introduction vignette in readme (#108)
openpharma/mmrm: README.Rmd

> The `mmrm` package implements mixed models for repeated measures (MMRM) in R
> based on Template Model Builder (TMB).

Reviewer: ```suggestion
The `mmrm` package implements mixed models for repeated measures (MMRM) in R.
```

Reviewer: then it is not confusing anybody, and we can elaborate on this more in the algorithm vignette

> *Main Features*  

Reviewer: ```suggestion
## Main Features
```
to use the same styling as below

* Responses are assumed normally distributed.
* Covariances: 
  * Structures: unstructured, Toeplitz, AR1, compound symmetry, and ante-dependence.

>   * Groups: shared covariance structure for all subjects, or shared by group.

Reviewer: ```suggestion
  * Groups: shared covariance structure for all subjects, or group specific covariance structures.
```

>   * Supports both time-homogeneous and heterogeneous structures.

Reviewer: ```suggestion
  * Variances: homogeneous or heterogeneous across time points.
```

* Hypothesis testing:
  * Least square means: `emmeans` package can be used with model outputs to obtain
least square means.

>   * Satterthwaite-adjusted one- and multi-dimensional contrasts. (future work: Kenward-Roger)

Reviewer: ```suggestion
  * Degrees of freedom adjustment: Satterthwaite-adjusted one- and multi-dimensional contrasts.
```

Reviewer: let's maybe take out the KR here
